src/codecs/openexr.rs

```diff
@@ -135,6 +135,23 @@ impl<'a, R: 'a + Read + Seek> ImageDecoder<'a> for OpenExrDecoder<R> {
         }
     }
 
+    fn set_limits(&mut self, limits: crate::io::Limits) -> ImageResult<()> {
```

Comment:
This can be left with the [default implementation](https://github.com/image-rs/image/blob/master/src/image.rs). We intentionally don't count the total bytes required for the output buffer towards the max_alloc limit because that is the caller's responsibility.